src/main/java/Step3/RacingCar.java

```diff
@@ -0,0 +1,31 @@
+package Step3;
+
+public class RacingCar {
+    private int pos;
```

Comment:
```suggestion
    private int position;
```
`pos`와 같은 축약어보다는 의미를 확실히 전달할 수 있는 네이밍이 좋을 것 같아요.